cli/pcluster/update/update_command.py

```diff
@@ -206,3 +207,18 @@ def _restore_desired_size(cfn_params, stack_name, scheduler):
     # If there are compute nodes running, preserve current desired capacity (if possible)
     if int(cfn_params["MinSize"]) <= desired_capacity <= int(cfn_params["MaxSize"]):
         cfn_params["DesiredSize"] = str(desired_capacity)
+
+
+def _check_cluster_models(base_config, target_config):
+    if target_config.cluster_model == ClusterModel.SIT:
+        if base_config.cluster_model == ClusterModel.HIT:
```

Comment:
What about having a single if:
`if target_config.cluster_model == ClusterModel.SIT and base_config.cluster_model == ClusterModel.HIT:`